.github/workflows/flutter.yaml

```
@@ -0,0 +1,37 @@
+        sudo apt-get update
+        sudo apt-get install -y libgtk-3-dev libglib2.0-dev
+    
+    - name: Set up Flutter
+      uses: subosito/flutter-action@v2
+      with:
+        flutter-version: '3.x'
+        channel: 'stable'
+    
+    - name: Get dependencies
+      run: flutter pub get
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)